internal/services/network/application_gateway_resource_test.go

```diff
@@ -1369,6 +1421,7 @@ resource "azurerm_application_gateway" "test" {
     http_listener_name         = local.listener_name
     backend_address_pool_name  = local.backend_address_pool_name
     backend_http_settings_name = local.http_setting_name
+    priority                   = 1
```

Comment:
Can we then make the `priority` to be `required` in the schema?

Reply:
add customdiff and doc